bindings/binding.cpp.tmpl

```diff
@@ -0,0 +1,16 @@
+// This file was autogenerated by the Chimera build.
```

Comment:
This entire file explicitly prefixes definitions with `chimera::binding::`. This is rather verbose - could we simply put everything inside `namespace` blocks instead?